### ISSUE TITLE
Parse error while updating repo

### ISSUE THREAD

**Issue by netlexer** [vodik/repose#53]
I've created a custom repo to hold packages from the AUR. Occasionally, an update will fail, e.g.:

```
$ repose -vf  custom                                 
repose: failed to parse files for jekyll-3.7.0-3/files
```

I'm not sure if it's the package or repose itself, jekyll is just an example. 

Updating from scratch (using --rebuild) works, though this is time consuming.

**Comment by AladW** [comment]
#52?

**Comment by netlexer** [comment]
I don't think so, as I didn't use repo-add at all to create the db. I just used 'repose -vf custom.db' in a directory with all the files.

**Comment by stefanhusmann** [comment]
Similar (?) thing here: failed to parse files for eggwm-qt5/desc.

**Comment by bemeurer** [comment]
I can confirm the same issue here with `repose: failed to parse desc for ttf-font-awesome-4-4.7.0-5/desc`.

How can one do the "rebuild" work-around?

**Comment by AladW** [comment]
`repose --rebuild`

**Comment by bemeurer** [comment]
@AladW `repose --rebuild` complains about lacking arguments. `repose --rebuild aur` (`aur` is the name of my local repo) does nothing, if you use the verbose flag it says `writing aur.db...`, but it finishes in a split second, and doesn't fix the issue.

**Comment by AladW** [comment]
I edited my comment, put perhaps `--rebuild` is also effected by this. You could remove the database and then run `repose` again...

Have you tried the `rust` branch?

**Comment by bemeurer** [comment]
I can confirm that `repose -vf custom --rebuild` does not fix the problem for me. It runs successfully, but trying `aursync chromaprint-git` fails with the same error I noted before. I'm going to try using the rust branch.

**Comment by bemeurer** [comment]
The `rust` branch does not build on the latest nightly. I'm going to open an issue and try fixing it.

**Comment by bemeurer** [comment]
@AladW Could you perhaps propose a workaround for this while it isn't fixed? It's just annoying since it renders `aursync` unusable on my system.

**Comment by bemeurer** [comment]
Apparently:

1. Removing the offending package
2. Running a rebuild as noted earlier
3. Installing the package through aursync

Seems to solve the issue!

**Comment by AladW** [comment]
Workaround from the aurutils perspective is too remove repose, so it falls back to repo-add. Not really something I enjoy saying on the repose tracker, but there it is...

**Comment by aphirst** [comment]
I'm getting the same issue, now. It's not clear what caused it, but once it's caused, no matter what I seem to try, I can't "fix" the existing db. I think it works fine again for a while if I nuke the db and make it totally from scratch, but it's not really a fix since eventually the problem comes back. I'll update this post if I can work anything else out.

**Comment by bemeurer** [comment]
@aphirst Use `aurutils-git` it no longer uses repose and works fine. Or wait until the new version gets released.
